### PR TITLE
Module 1 [Block 1- Day 2] 

### PR DESCRIPTION
**Pull request by porag-m06**
## Hello Microverse Project
Completed requirements are: 
- Created "Hello word" repo.
- Created branch "add-homepage"
- Added .gitignore file and added node-modules/ & test.md to it.
- Added and modified README form template
- Added index.html with the required header
- Added styles.css with text color modification of the header
- Added test.md flle

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,121 @@
...
+### Run tests
+N/A
+
+### Deployment
+N/A
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
```
Comment (mukhammadrizooff) [inline README.md]: - [ ] [REQUIRED] It would be descriptive if you remove `-s` end of `Authors` word 👍🏼

File: README.md [hunk 2]
```diff
@@ -1 +1,121 @@
...
+  - Yes, you can share this project for any educational purposes. 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```
Comment (mukhammadrizooff) [inline README.md:120]: - [ ] [REQUIRED] Kindly your `MIT` license is not working, please fix it 🙏🏼

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @porag-m06, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct GitHub Flow ✔️ 
- Good implementations ✔️ 
- Descriptive `PR` summary and `Commit` history ✔️ 

## Required Changes ♻️
- [ ] [REQUIRED] Good job so far! The linter result should be `green`. `AKA`: Linter should report `NO` errors, please fix your linter errors 🙏🏼 

image


_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
### Hi @porag-m06, 

# Status: Approved ✔️
Excellent job on making the changes requested by the previous code reviewer. Keep it up 👏

Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉
congratulations
>
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Followed Github flow :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and description :heavy_check_mark:
 
## [Optional] suggestions:
* Nothing to mention.
 
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
 
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**